### PR TITLE
morphoi: explicitly depend on highfive to solve build issues with mpi.

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: var/spack/repos/builtin/packages/morphio/package.py [hunk 1]
```diff
@@ -25,10 +25,15 @@ class Morphio(CMakePackage):
     depends_on('cmake@3.2:', type='build')
     depends_on('hdf5~mpi', when='~mpi')
     depends_on('hdf5+mpi', when='+mpi')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/morphio/package.py]: Do we need hdf5 here then? I assume morphio doesn’t use h5 directly but via highfive only

Comment (matz-e) [reply var/spack/repos/builtin/packages/morphio/package.py]: I guess technically not. It shipped it's own version of HighFive before…

**Comment by ferdonline** [comment]
For a moment I was afraid there's a new morph* package (typo in PR name)

**Comment by pramodk** [comment]
By the way, this is going to rebuild all simulation stack!

**Comment by matz-e** [comment]
It's going to rebuild, might as well update highfive, too… that's needed to unblock synapse-tool.

**Comment by matz-e** [comment]
Ping! @pramodk ?

**Comment by matz-e** [comment]
Outdated.

**Pull request closed without merging**